kernel/sortPolyFlags.mli

```diff
@@ -0,0 +1,22 @@
+type t
+(* Set of flags for sort polymorphism, universe level polymorphism and cumulativity *)
+
+(* Raises an error if sort_poly or cumulative is true but poly isn't *)
+val make : level_polymorphic:bool -> sort_polymorphic:bool -> cumulative:bool -> t
+
+(* All false *)
+val default : t
+
+(* Only set the universe polymorphism flag *)
+val of_poly : bool -> t
+
+val sort_polymorphic : t -> bool
+val level_polymorphic : t -> bool
+val cumulative : t -> bool
+
+(* Alias of level_polymorphic *)
+val is_polymorphic : t -> bool
```

Comment:
Why have such an ambiguous alias?
I could understand, if it were supposed to mean the disjunction of all fields, with the invariant that if any of them is true then `level` is true. (Then this would have to be made explicit)

Reply:
The invariant is indeed that if any of the fields is true then level_polymorphic is true, so is_polymorphic tells us if we're making a poly or mono declaration. It's used to decide e.g. if we're making a polymorphic entry or a monomorphic one in `UState.univ_entry`. Maybe it's not such a good idea to have this alias, we can just test `is_level_polymorphic` instead. The idea was to have the nomenclature encompass the case where we'll have sort variables as well, for cases like this one: https://github.com/rocq-prover/rocq/pull/21419/changes#diff-bed94d796d77bc3c3710706202a044e2c2affce34257337ec06dd4ce632b4fd1R54 but maybe its an XY here and comPrimitive should just not rely on a test if a context is empty to decide the status of the flags.

Reply:
I'm not sure either whether it's an XY; but this sounds fine for a semantics. This should however be made completely clear in the docstring: it's not just an alias, it's a test on whether any field is true (and the fact that you only need to test one is internal details).